packages/scandipwa/src/component/ProductCard/ProductCard.component.js

```diff
@@ -129,7 +135,7 @@ export class ProductCard extends PureComponent {
                   elem: 'PriceBadge'
               } }
             >
-                { __('As Low as') }
+                { __(label) }
```

Comment:
NO.

This is not a correct usage of i18n plugin. This way, the value cannot be extracted into the translations JSON.
You must've been warned about this by the i18n plugin. Please don't ignore its messages.
The only appropriate usage for the `__(...)` function is passing string to it directly. 
I.e. you need to do it above: `__('Starting from')`, `__('As Low as')`

Reply:
Moved above